src/sdjwt.ts

```diff
@@ -188,9 +206,9 @@ export const listKeys = (obj: any, prefix: string = '') => {
 
 export const pack = async <T extends object>(
   claims: T,
-  disclosureFrame?: DisclosureFrame<T>,
-  hasher: Hasher = hash,
-  saltGenerator: SaltGenerator = generateSalt,
+  disclosureFrame: DisclosureFrame<T> | undefined,
```

Comment:
```suggestion
  disclosureFrame?: DisclosureFrame<T>,
```

Reply:
It can't be optional because its not the last parameter.